sample/src/main/java/com/zhihu/matisse/sample/FrescoImageEngine.java

```diff
@@ -0,0 +1,190 @@
+package com.zhihu.matisse.sample;
+
```

Comment:
why put this file in sample directory?It had better be put in com.zhihu.matisse.engine.impl directory

Reply:
This only provides some life-cycle external dependencies, so that you can customize the ImageEngine to your needs, and the FrescoImageEngine is just a sample